### PR TITLE
Adding Sentinel-2 L1C inventory DDL

### PR DESCRIPTION
**Pull request by jflasher**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jflasher** [comment]
@zflamig would you be able to check this over?

File: datasets/sentinel-2.yaml [hunk 1]
```diff
@@ -142,3 +142,34 @@ DataAtWork:
...
 
+Glue-DDL:
+- Name: s2_l1c_inventory
+  StorageDescriptor:
+    InputFormat: org.apache.hadoop.hive.ql.io.SymlinkTextInputFormat
+    OutputFormat: org.apache.hadoop.hive.ql.io.IgnoreKeyTextOutputFormat
+    StoredAsSubDirectories: True
+    Location: s3://sentinel-inventory/sentinel-s2-l1c/inventory-orc/hive/
+    SerdeInfo:
+      Parameters:
```
Comment (zflamig) [inline datasets/sentinel-2.yaml:153]: Don't need this

File: datasets/sentinel-2.yaml [hunk 2]
```diff
@@ -142,3 +142,34 @@ DataAtWork:
...
+      Type: string
+    - Name: key
+      Type: string
+    - Name: size
+      Type: bigint
+    - Name: last_modified_date
+      Type: bigint
+    Parameters:
+        projection.dt.format: yyyy-MM-dd-HH
+        projection.dt.interval: 1
```
Comment (zflamig) [inline datasets/sentinel-2.yaml:167]: quote this, only takes strings

File: datasets/sentinel-2.yaml [hunk 3]
```diff
@@ -142,3 +142,34 @@ DataAtWork:
...
+        projection.dt.interval.unit: HOURS
+        projection.dt.range: "2021-09-01-00,NOW"
+        projection.dt.type: date
+        projection.enabled: True
```
Comment (zflamig) [inline datasets/sentinel-2.yaml:171]: quote this only strings

File: datasets/sentinel-2.yaml [hunk 4]
```diff
@@ -142,3 +142,34 @@ DataAtWork:
...
+    Columns:
+    - Name: bucket
+      Type: string
+    - Name: key
+      Type: string
+    - Name: size
+      Type: bigint
+    - Name: last_modified_date
+      Type: bigint
+    Parameters:
```
Comment (zflamig) [inline datasets/sentinel-2.yaml:165]: outdent this one level

File: datasets/sentinel-2.yaml [hunk 5]
```diff
@@ -142,3 +142,34 @@ DataAtWork:
...
+        projection.dt.format: yyyy-MM-dd-HH
+        projection.dt.interval: 1
+        projection.dt.interval.unit: HOURS
+        projection.dt.range: "2021-09-01-00,NOW"
+        projection.dt.type: date
+        projection.enabled: True
+        storage.location.template:
```
Comment (zflamig) [inline datasets/sentinel-2.yaml:172]: s3://sentinel-inventory/sentinel-s2-l1c/inventory-orc/hive/dt=${dt}-00 here

**Comment by jflasher** [comment]
Thanks @zflamig, I think I hit all the comments.

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 673dd919e2afce8330a02211e0002a8484c01153
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=52316b46-0d8a-4d35-8da6-739ee7855bc0%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by zflamig** [comment]
Works great, thanks!

**Pull request merged**